packages/ruby/README.md

```diff
@@ -11,20 +11,37 @@ Track your API metrics within ReadMe.
 `Readme::Metrics` is a Rack middleware and is compatible with all Rack-based
 apps, including Rails.
 
+When configuring the middleware, you must provide a block to tell the
+middleware how to get values for the current user. These may be values taken
+from the environment, or you may hardcode them.
+
 ### Rails
 
 ```ruby
 # application.rb
 require "readme/metrics"
 
-config.middleware.use Readme::Metrics, "YOUR_API_KEY"
+config.middleware.use Readme::Metrics, "YOUR_API_KEY"do |env|
+  {
+    id: env['CURRENT_USER'].id,
+    label: env['CURRENT_USER'].full_name,
+    email: env['CURRENT_USER'].email
+  }
+end
+
 ```
 
 ### Rack::Builder
 
 ```ruby
 Rack::Builder.new do |builder|
-  builder.use Readme::Metrics, "YOUR_API_KEY"
+  builder.use Readme::Metrics, "YOUR_API_KEY" do |env|
+    {
+      id: "my_application_id"
+      label: "My Application",
+      email: "my.application@example.com"
```

Comment:
Is there a situation where a client would want to hardcode this to the application name?

Reply:
Yep, I believe so. Based on the wording of the documentation for building an SDK, these values can either be for a unique user or values for the application itself.

https://docs.readme.com/metrics/docs/building-api-metrics-middleware